docs/src/main/sphinx/admin/properties-http-client.md

```diff
@@ -15,6 +15,7 @@ oauth2-jwk.http-client.connect-timeout=10s
 
 The following prefixes are supported:
 
+- `ai` for [](/functions/ai)
```

Comment:
This doesn't belong here, as the AI Functions are a connector, so this would need to go in the catalog config. The link back to this page should be sufficient.

Reply:
ok